lms/djangoapps/branding/api.py

```diff
@@ -569,22 +568,43 @@ def get_logo_url(is_secure=True):
         is_secure (bool): If true, use HTTPS as the protocol.
     """
 
-    # if the configuration has an overide value for the logo_image_url
+    # if the configuration has an override value for the logo_image_url
     # let's use that
-    image_url = configuration_helpers.get_value('logo_image_url')
-    if image_url:
-        return _absolute_url_staticfile(
-            is_secure=is_secure,
-            name=image_url,
-        )
+    logo_url = settings.LOGO_URL
+    default_local_path = 'images/logo.png'
+    logo_image_path = configuration_helpers.get_value('logo_image_url')
```

Comment:
Code readability: Can you rename this variable so it's clear that this value is coming from site-config?  Maybe `logo_url_from_site_config`?